test/utils/runners.go

```diff
@@ -1433,7 +1436,7 @@ func CreatePodWithPersistentVolume(ctx context.Context, client clientset.Interfa
 				},
 			},
 		}
-		if err := makeCreatePod(client, namespace, pod); err != nil {
+		if err := makeCreatePod(client, namespace, pod.DeepCopy()); err != nil {
```

Comment:
Do we need to deepCopy here, we already did this in L:1428

Reply:
You are right, that shouldn't be necessary here.